src/ui/connection_profiles.ui

```diff
@@ -2454,6 +2425,35 @@
               </property>
              </widget>
             </item>
+            <item row="3" column="0" colspan="2">
+             <widget class="QCheckBox" name="autologin_checkBox">
+              <property name="sizePolicy">
+               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
+                <horstretch>0</horstretch>
+                <verstretch>0</verstretch>
+               </sizepolicy>
+              </property>
+              <property name="toolTip">
+               <string>With this enabled, Mudlet will automatically start and connect on this profile when it is launched</string>
+              </property>
+              <property name="layoutDirection">
+               <enum>Qt::LeftToRight</enum>
+              </property>
+              <property name="text">
+               <string extracomment="This option causes this profile to be automatically LOADED when the Mudlet application is started; the internal name is functionally incorrect!">Open profile on Mudlet start</string>
```

Comment:
Do players see this text at all?

Reply:
Nope - it is internal documentation - I think it is visible to translators - to try and explain that the control name is misleading.